cmd/podman/root.go

```diff
@@ -171,6 +171,13 @@ func persistentPreRunE(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if !registry.IsRemote() {
+		err := registry.ContainerEngine().EarlyProcessSetup(registry.Context())
```

Comment:
This might be better as part of `cobra.OnInitialize()` - we're already configuring logging there

Reply:
(Most notably, removes the need for a public function to do the work)

Reply:
are we able there to distinguish between the remote and the ABI case?

Reply:
well probably it makes no sense to distinguish, we can run this in both cases